ui/webui/resources/css/paper_button_style.css

```diff
@@ -7,73 +7,53 @@
 /* Refer to:
  * ui/webui/resources/cr_elements/paper_button_style_css.html
  * for styles to override.
- *
- * Translation of variants from Paper to Brave:
- * action-button = Primary (solid fill bg, white fg)
- * default = Secondary (outline color bg, white bg, color fg)
- *
  */
 
+/* light theme */
 paper-button {
-  --brave-hover-color: #e7e7ec;
-  --brave-active-color: white;
-  --paper-button-ink-color: #242536 !important;
-
-  /* normal button */
-  --text-color: #242536 !important;
-  --border-color: var(--text-color) !important;
-  --hover-bg-color: var(--brave-hover-color) !important;
-  --hover-border-color: var(--text-color) !important;
-  --active-bg: var(--brave-active-color) !important;
-
-  /* action button */
-  --flat-text-color-action: white !important;
-  --flat-bg-action: var(--text-color) !important;
-  --hover-bg-action: var(--flat-bg-action) !important;
-  font-family: Poppins;
-  /* NOTE(petemill): some pages override default line-height for text,
-     but that shouldn't flow to buttons. It affects text centering. */
-  line-height: 1 !important;
-
-  border-radius: 20px !important;
-  border-color: currentColor !important;
-
-  --flat-focus-shadow-color: rgba(255, 69, 48, .4)  !important;
+  --type-color: #3b3e4f; /* grey800 */
```

Comment:
Is this a variable name we're making here? If so please prefix with brave- otherwise we may conflict with chromium (these variable names are global).
Same for `default-border`.

Anything without a prefix should be something defined in the chromium webui codebase that we want to override.
Anything with a prefix like `paper-button-` will be something defined in a specific paper (or other) library component that we want to override.

Apologies that there's no docs yet for this.